yoti_test.go

```diff
@@ -1018,24 +1018,6 @@ func TestAnchorParser_YotiAdmin(t *testing.T) {
 	AssertServerCertSerialNo(t, "256616937783084706710155170893983549581", actualSerialNo)
 }
 
-func TestAnchors_Unknown(t *testing.T) {
```

Comment:
Since anchor type is unexported now, we can't create this unknown anchor in the way that we're doing in this test. I spoke with Mike and have created https://lampkicking.atlassian.net/browse/SDK-686 to reimplement this test once we have a mocking framework. Drayton also mentioned Core had some testing tools, which might solve this.